### PR TITLE
Remove *I overloads and move scopes

### PR DESCRIPTION
**Pull request by JeremyKuhne**
Int overloads just cast to float, removing non-array ones to cut back on interop code.

- Fix some missing KeepAlives
- Move error handling for GdiPlus to Core
- Move some scopes down and convert RegionScope to use IPointer

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/10640)

### CODE REVIEW

**Comment by JeremyKuhne** [comment]
This looks bigger than it is as it adds loc files to the Core assembly.

**Comment by JeremyKuhne** [comment]
There is some intermittent packaging issue with the resources that I'm exploring, but otherwise this is good for review.

File: src/System.Private.Windows.Core/src/Windows/Win32/Graphics/Gdi/RegionScope.cs [hunk 1]
```diff
@@ -15,7 +16,7 @@ namespace Windows.Win32.Graphics.Gdi;
 ///  </para>
 /// </remarks>
 #if DEBUG
-internal class RegionScope : DisposalTracking.Tracker, IDisposable
+internal unsafe class RegionScope : DisposalTracking.Tracker, IDisposable
 #else
 internal ref struct RegionScope
```
Comment (lonitra) [inline src/System.Private.Windows.Core/src/Windows/Win32/Graphics/Gdi/RegionScope.cs:21]: Think we may also need unsafe here
```suggestion
internal unsafe ref struct RegionScope
```

**Comment by JeremyKuhne** [comment]
The resources I copied were from `System.Drawing.Common`, which aren't currently localized.

**Review by lonitra — APPROVED** [review]
🚀

**Pull request merged**